pkgs/development/libraries/ncurses/default.nix

```diff
@@ -46,6 +46,9 @@ stdenv.mkDerivation rec {
   buildInputs = lib.optional (mouseSupport && stdenv.isLinux) gpm;
 
   preConfigure = ''
+    # These paths end up in the default lookup chain.
+    export TERMINFO_DIRS=/etc/terminfo
+
     export PKG_CONFIG_LIBDIR="$dev/lib/pkgconfig"
```

Comment:
Yeah, that seems to be a recurring thing. I would like people to fix things when they are glaring in my face, and preferably also before they do a pull request for new features. I.e., to go over whatever wrong things the person before you has done and to fix it.

I.e., instead of arguing over it, wouldn't it be faster to just fix it and amend a commit? The alternative would be for me to create a PR specifically to fix whatever stupid quoting people do.